app/Http/Transformers/UserTransformer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use League\Fractal\TransformerAbstract;
+
+class UserTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $user
+     * @return array
+     */
+    public function transform($user)
+    {
+        return [
+            'first_name' => $user->first_name,
```

Comment:
@DFurnes I double checked with @ngjo and currently `first_name` is all that is requested. Northstar ID is included in the signup part of the response anyway so that will always be included.